contrib/debian/fwupd-qubes-vm-whonix.postrm

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm -rf /home/user/.cache/fwupd
```

Comment:
Likewise here `user` is hardcoded to a fixed name?  Doesn't there need to be some sort of lookup process for the correct ID and group?

Reply:
I think the same safety check I suggested using `getent passwd user` should apply here.

Furthermore, Debian packages aren't supposed to be removing a user's data in the postinst unless called with `purge` as an argument.  So probably you should be checking for that too.

Reply:
I've added proper checks.